impl/src/main/java/org/ehcache/internal/store/tiering/CacheStore.java

```diff
@@ -312,6 +313,12 @@ public void disableStoreEventNotifications() {
     }
   }
 
+  @Override
+  public CacheConfigurationChangeListener getConfigurationListener() {
+//    throw new UnsupportedOperationException("TODO Implement me!");
+    return null;
```

Comment:
Also, when doing partial commits, they should not result in causing issues , having no-ops is acceptable but ending up with exceptions is not.